requirements.txt

```diff
@@ -40,8 +40,10 @@ pytest-mock==1.1
 # Upstream url: https://github.com/python/typing
 typing==3.6.4
 
+# A common package that holds the models deifnition and schemas that are used
+# accross different amundsen repositories.
+amundsen-common>=0.3.1,<1.0
 
-amundsen-common>=0.3.0,<1.0
```

Comment:
won't it auto pick up the upper bound?

Reply:
To force update to the newer version in any case, since the previous version had errors.